arithmetization/src/main/java/net/consensys/linea/zktracer/module/exp/ModExpLogChunk.java

```diff
@@ -92,83 +93,9 @@ public static ModExpLogChunk fromExpLogCall(final Wcp wcp, final ModExpLogCall c
     return modExpLogChunk;
   }
 
-  public static ModExpLogChunk fromFrame(final Wcp wcp, final MessageFrame frame) {
```

Comment:
Where did this Chunk builder function go ?

Reply:
deleted, it was never called. The hub fully triggers exp (as before it was triggered by the hub AND by tracePreOpcode() for exp)